tests/python/test_oop.py

```diff
@@ -14,20 +14,11 @@ def __init__(self, n, m):
     @ti.classfunc
     def inc(self, i, j):
       self.val[i, j] += i * j
-      
-    @ti.func
```

Comment:
Actually one thing that would make programmers' life easier is to merge `ti.func` and `ti.classfunc` together, and `ti.kernel` and `ti.classkernel` together. Then they don't have to use different decorators for kernels/functions inside/outside classes :-)

Reply:
I managed to make `ti.func` work in classes, yet I failed to make `ti.kernel` work in classes. It'll be great if you can manage to do that :-) Then Taichi programmers will have one less concept to learn (`ti.classkernel`).